src/components/rockets/Nav.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import logo from '../assets/logo.png';
+import '../styles/Nav.css';
+
+function Nav() {
+  const addLine = (e) => {
+    e.target.classList.toggle('click');
+  };
+  return (
+    <section className="header-navigation">
+      <div className="header">
+        <img src={logo} alt="logo" className="logo" />
+        <h1 className="proj-name">Space Traveler`s Hub</h1>
+      </div>
+      <ul className="navigation">
+        <li className="navigation-item"><button className="navigation-item rockets" type="button" onClick={addLine}>Rockets</button></li>
+        <li className="navigation-item"><button className="navigation-item missions" type="button" onClick={addLine}>Missions</button></li>
+        <line className="border" />
+        <li className="navigation-item"><button className="navigation-item profile" type="button" onClick={addLine}>My Profile</button></li>
+      </ul>
```

Comment:
@kamranahmad123 Hello Kamran, Good job so far, I see everything you have done nicely. I have a small suggestion for you to move further. Please use <NavLink></NavLink> here instead of the button, so the page navigation works correctly as expected and the active page link item shows differently.